Makefile

```diff
@@ -67,12 +72,15 @@ $(PLATFORMS): ensure-version check test
 	@ echo "==> Building $(OS) distribution"
 	@ mkdir -p $(BIN)/$(OS)/$(ARCH)
 	@ mkdir -p $(DIST)
+
 	CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build $(BUILDARGS) -o $(BIN)/$(OS)/$(ARCH)/$(NAME)
 	cp -f $(BIN)/$(OS)/$(ARCH)/$(NAME) $(DIST)/$(NAME)-$(OS)-$(ARCH)
 
+	@ $(SHA256_CMD) $(DIST)/$(NAME)-$(OS)-$(ARCH) | awk '{$$2="$(NAME)-$(OS)-$(ARCH)"; print $$0}' >> $(DIST)/$(NAME).SHA256SUMS
```

Comment:
I'm can't say for `sha256sum`, but for `shasum` on macOS, the generated file can't be used to check the binaries, e.g.

```sh
$ cd dist
$ shasum -a 256 -c terrahelp.SHA256SUMS
shasum: terrahelp.SHA256SUMS: no properly formatted SHA checksum lines found
```

If you use the direct output from `shasum`, it works:
```
$ cd dist
$ shasum -a 256 terrahelp-* > terrahelp.SHA256SUMS
$ shasum -a 256 -c terrahelp.SHA256SUMS
terrahelp-darwin-amd64: OK
terrahelp-linux-amd64: OK
```

Perhaps it's something in the `awk` expression is amiss?